test/test.geocoder.js

```diff
@@ -513,5 +513,54 @@ test('geocoder', function(tt) {
     t.end();
   });
 
+  tt.test('options.collapsed=true', function(t) {
```

Comment:
nitpick: these tests look good, but should we move them to `test.ui.js` since they generally handle the testing of UI elements?